intranethome/tests.py

```diff
@@ -1,3 +1,32 @@
-from django.test import TestCase
-
+from django.test import SimpleTestCase
+from intranethome.views import uniforming_into_list_of_dict, format_splitting
 # Create your tests here.
+
+class test_mail_aliases_view(SimpleTestCase):
```

Comment:
These tests are nice! I would like to see more here though. One idea is to test that the error case handles as expected.